airbyte-server/src/main/java/io/airbyte/server/handlers/SourceHandler.java

```diff
@@ -170,10 +170,7 @@ public void deleteSource(final SourceRead source)
     }
 
     final ConnectorSpecification spec = getSpecFromSourceId(source.getSourceId());
-    validateSource(spec, source.getConnectionConfiguration());
-
     final var fullConfig = configRepository.getSourceConnectionWithSecrets(source.getSourceId()).getConfiguration();
-    validateSource(spec, fullConfig);
```

Comment:
@jrhizor i think deleting both of these is the right thing to do, but curious if to hear if you think there's any value in keeping the second one.